src/types/modules/react-native-onyx.d.ts

```diff
@@ -1,9 +1,9 @@
-import type {OnyxCollectionKey, OnyxKey, OnyxValues} from '@src/ONYXKEYS';
+import type {OnyxCollectionKey, OnyxFormDraftKey, OnyxFormKey, OnyxValueKey, OnyxValues} from '@src/ONYXKEYS';
 
 declare module 'react-native-onyx' {
     // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
     interface CustomTypeOptions {
-        keys: OnyxKey;
+        keys: OnyxValueKey | OnyxFormKey | OnyxFormDraftKey;
```

Comment:
I can't use Exclude<OnyxKey, OnyxCollectionKey> because of `reimbursementAccount` onyx keys 😮‍💨

Reply:
I think it's better to leave as is now, such TODOs are never resolved and just pollute the code 😅

Reply:
leaving TODOs in the code is a bad practice (at Expensify, anyways...). It's better to just create a follow-up GitHub issue